content/en/certifications/_index.md

```diff
@@ -9,5 +9,5 @@ images: []
 weight: 100
 layout: default
 ---
-[LRA 1.0 Compatible Certification Request (CCR)](lra/microprofile-lra-1.0/index.html)
```

Comment:
We need to maintain a running list of all certified versions that we're willing to "support" and since this is a community project it will do no harm to list previous certifications.

Reply:
I have just updated it. Thanks